testament/specs.nim

```diff
@@ -289,6 +289,16 @@ proc parseSpecifiedTargets*(value: string): set[SpecifiedTarget] =
       else: raise newException(ValueError,
                                "invalid target specificatoin: '$#'" % v)
 
+proc splitKey*(key: string): tuple[key, sub: string] =
```

Comment:
did you already consider using section start/end, IIRC this is all based on `parsecfg` so that should be available to us?

```
[knownIssues]
target.js: "lorem ipsum ..."
```

Having said that, this might be a better evolution once we support more kinds of known issues.